core/src/test/java/com/sequenceiq/cloudbreak/FlowPayloadConstructorTest.java

```diff
@@ -0,0 +1,40 @@
+package com.sequenceiq.cloudbreak;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Modifier;
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+import org.reflections.ReflectionUtils;
+import org.reflections.Reflections;
+import org.reflections.scanners.SubTypesScanner;
+
+import com.sequenceiq.cloudbreak.common.event.Acceptable;
+
+class FlowPayloadConstructorTest {
+
+    @Test
+    void consturctorAccessible() {
+        Reflections reflections = new Reflections("com.sequenceiq", new SubTypesScanner(true));
+        Set<Class<? extends Acceptable>> eventClasses = reflections.getSubTypesOf(Acceptable.class);
+        eventClasses.stream()
+                .filter(c -> !c.isInterface() && !isAbstract(c) && !c.isAnonymousClass() && !c.isLocalClass() && !c.isMemberClass())
+                .filter(c -> !c.getName().endsWith("Test"))
+                .forEach(this::checkForConsturctor);
+    }
+
+    private void checkForConsturctor(Class<? extends Acceptable> clazz) {
```

Comment:
Typo: Consturctor -> Constructor
Please fix this in all the test class copies below as well.

Reply:
🤣